packages/datepicker-react/src/internal/Calendar.tsx

```diff
@@ -299,6 +300,7 @@ export const Calendar = forwardRef<HTMLDivElement, CalendarProps>((props, ref) =
                         />
                         <NativeSelect
                             className="jkl-calendar__month-selector"
+                            selectClassName="jkl-calendar__year-selector-select"
```

Comment:
Oops, det skulle vært her @BjorneOmaFremtind – litt rask med copypastaen
```suggestion
                            selectClassName="jkl-calendar__month-selector-select"
```